CONTRIBUTING.md

```diff
@@ -109,6 +110,21 @@ You can add a new community event by creating a new markdown file in [data/meetu
 
 To update the list of Featured Packages in the Packages page, you can update [data/packages.yml](data/packages.yml)
 
+### <a name="ocaml-changelog"></a> OCaml Changelog
+
+The [OCaml Changelog](https://ocaml.org/changelog) is a feed of the latest releases and feature highlights for official OCaml projects. As of today, it features the following projects:
+
+- [The OCaml compiler](https://github.com/ocaml/ocaml)
+- [The OCaml Platform tools](https://ocaml.org/docs/platform)
+- [Ppxlib](https://github.com/ocaml-ppx/ppxlib)
+- [Omd](https://github.com/ocaml/omd) (that has been deprecated in favour of Ppxlib)
```

Comment:
Omd has been deprecated in favor of Ppxlib? :thinking: That's what I read here. I am confused both on whether that's true and also what this means.

Reply:
Sorry, should be [OMP](https://github.com/ocaml-ppx/ocaml-migrate-parsetree), not Omd.

Reply:
Ok good, that makes a lot more sense

Reply:
Fixed, thanks for spotting it.